lib/java/src/main/java/org/apache/thrift/TEnumHelper.java

```diff
@@ -38,11 +41,7 @@ public static TEnum getByValue(Class<? extends TEnum> enumClass, int value) {
     try {
       Method method = enumClass.getMethod("findByValue", int.class);
       return (TEnum) method.invoke(null, value);
-    } catch (NoSuchMethodException nsme) {
-      return null;
-    } catch (IllegalAccessException iae) {
-      return null;
-    } catch (InvocationTargetException ite) {
+    } catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException nsme) {
```

Comment:
I think these can be replaced with:

```suggestion
    } catch (ReflectiveOperationException nsme) {
```